app/grandchallenge/hanging_protocols/forms.py

```diff
@@ -88,6 +88,14 @@ def clean_json(self):
 class ViewContentMixin:
     def clean_view_content(self):
         mapping = self.cleaned_data["view_content"] or {}
+        if not isinstance(mapping, dict):
+            self.add_error(
+                error=f"Value {str(mapping)} is not valid. "
+                "Should be of type `object`.",
+                field="view_content",
+            )
+            # return, because the ci check assumes the value is a dict
+            return mapping

```

Comment:
Why return? Shouldn't a Validation error be raised here? I thought `add_error` should only be used on the model clean method, not individual fields.

Reply:
Looked it up and you're absolutely right. I'll update the form and the test.